UKHomeOffice/sgmr-service
# NMSW-1380 Merge component folders

### AC / Description of changes

Tidy up files ahead of new feature work
- merge component folders

### To Test

- run tests they should pass
- sign into app
- add a person
- delete a person
- > you should be taken to people list with a confirmation of person deleted banner
- cancel a voyage plan
- > you should be taken to dashboard with a confirmation of delete banner

### Notes

---

**Remember to merge to feature branches not `master`.**
